packages/backend/package.json

```diff
@@ -21,17 +21,17 @@
     "test:e2e": "SEED_ROUND_EVENT_HITS=1 cross-env TZ=UTC jest --runInBand --config ./test/jest-e2e.json"
   },
   "dependencies": {
-    "@nestjs/common": "^9.4.0",
-    "@nestjs/config": "^2.3.1",
-    "@nestjs/core": "^9.4.0",
+    "@nestjs/common": "^10.2.2",
+    "@nestjs/config": "^3.0.0",
+    "@nestjs/core": "^10.2.2",
     "@nestjs/jwt": "^10.0.3",
-    "@nestjs/passport": "^9.0.3",
-    "@nestjs/platform-express": "^9.4.0",
-    "@nestjs/platform-ws": "^9.4.0",
+    "@nestjs/passport": "^10.0.1",
+    "@nestjs/platform-express": "^10.2.2",
+    "@nestjs/platform-ws": "^10.2.2",
     "@nestjs/schedule": "^3.0.2",
-    "@nestjs/swagger": "^6.3.0",
+    "@nestjs/swagger": "^7.1.10",
     "@nestjs/throttler": "^4.0.0",
-    "@nestjs/websockets": "^9.4.0",
+    "@nestjs/websockets": "^10.2.2",
```

Comment:
I made the upgrade like a gros bourrin, I trust the CI 😄

Reply:
CI is the truth 🙏🏻